yarn.lock

```diff
@@ -2222,27 +2222,6 @@
   dependencies:
     "@types/yargs-parser" "*"
 
-"@walletconnect/browser-utils@^1.7.1":
```

Comment:
FYI: The changes in this file are not mine. They're auto generated whenever I run `make test`. I believe somebody merged into `develop` without some yarn lock changes.